### ISSUE TITLE
Add Rocky And Alma Linux Scanner Support

### ISSUE THREAD

**Issue by msierks** [stackrox/stackrox#1727]
Hello!
I bet there are a few who'd like this, it'd be great to support Rocky and Alma Linux (or at least partially). Specifically to have these image releases and there components identified by the scanner.

**Comment by Maddosaurus** [comment]
Hey @msierks and thank you for your request!
We're currently discussing this in the engineering team and will come back to you once we have an update!

**Comment by RTann** [comment]
We have discussed this, and support for these two OSes will be community-driven at this time. That means we will likely not work towards developing this support nor fixing bugs, but we will happily review any related PRs

**Comment by msierks** [comment]
This sounds reasonable to me.

**Comment by msierks** [comment]
Sorry, I made an assumption here. It's a little unclear, will PRs related to Rocky and Alma support be accepted/merged.

**Comment by msugakov** [comment]
Hi @msierks 
I suppose you mean <https://github.com/stackrox/scanner/pull/745> and <https://github.com/stackrox/scanner/pull/748>, correct? I double-checked with the team and in general we're open and grateful to integrate Rocky and Alma support as community contribution. It looks like @RTann and colleagues responsible for the Scanner component may not have been tagged to review the recent changes in these PRs. I tagged Ross in both PRs and so he should get notified.
In either case, feel free to follow-up if you see PRs not getting attention.
That said, we have certain code standards and may open PR comments which may have to be addressed before we consider that PR can be merged. I think such thing is natural to expect from many open source projects these days. Please have this in mind and try to respond all comments in PRs or be explicit when the comment is unclear. This should speed up review and approval.
Thanks!

**Comment by msierks** [comment]
@msugakov Yes, those are the PRs I was referring to. Thanks for clearing that up.
